src/components/Footer/Footer.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { makeStyles } from '@mui/styles';
+import {
+  AppBar, Box, Container, Toolbar, IconButton, Menu, MenuItem, Button, Typography
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { useHistory } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import logo from 'assets/mitre-logo.svg';
+
+const useStyles = makeStyles((theme) => ({
+  footer: {
+    color: theme.palette.primary.main,
+    backgroundColor: theme.palette.common.grayLight,
+    position: 'static',
+    marginTop: '2em',
+  },
+  link: {
+    fontWeight: 700,
+  },
+  logo: {
+    objectFit: 'contain',
+    height: '3.5em',
+  }
+}));
+
+const pages = ['FAQ', 'Privacy'];
+
+const Footer = () => {
+  const classes = useStyles();
+  const history = useHistory();
+  const { t } = useTranslation();
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = (page) => {
+    setAnchorElNav(null);
+    history.push(`/${page}`);
```

Comment:
While viewing the nav menu in mobile view, when you click anywhere on the screen besides the "FAQ" and "Privacy" buttons, `page` here becomes an object, which brings you to undefined route / blank screen.

Reply:
Good catch, fixed in latest commit.

Reply:
Fixed!